common/app/views/fragments/page/email/head.scala.html

```diff
@@ -2,6 +2,14 @@
 @import common.{LinkTo, CanonicalLink}
 @import views.support.EmailHelpers.Images.footerG
 <head>
+    <!--[if gte mso 9]>
```

Comment:
Could you please add the [link to the reference](https://litmus.com/community/snippets/112-outlook-2013-120dpi-make-images-scale-properly) directly in the code here.

Reply:
Thank you @mchv for your comment. I have just added a comment with a link to the reference. It would be great if you could review when you have a chance.